styles/styles.css

```
@@ -234,6 +254,10 @@ main {
     line-height: 40px;
     color: #000000;
     text-decoration: none;
+    margin-top: 45px;
+    margin-left: 448,5px;
+    margin-right: 420,5px;
+    margin-bottom: 35px;
```

```suggestion
    margin-top: 45px;
    margin-bottom: 35px;
```
Тут два варианта: либо это паддинги секции либо ширина дочерних и margin auto

В css нет запятых в единицах изменения, тут ставят точки. 
**Но принято округлять до целых чисел.**